views/AppView.js

```diff
@@ -0,0 +1,64 @@
+var AppView = Backbone.View.extend({
+  el: $('body'),
+
+  events: {
+    'click .search': 'handleSearch',
+    'click .view-video': 'viewVideo'
+  },
+
+  mainVideo: null,
+
+  initialize: function () {
+    this.listenTo(this.model.get('videos'), 'add', this.renderVideo);
+    this.listenTo(this.model.get('videos'), 'add', this.setCurrentVideo);
+    this.listenTo(this.model, 'change:show_video', this.renderPage);
+    this.listenTo(this.model, 'change:current_video', this.renderMainVideo);
+    this.$searchInput = this.$('#search-query');
+    this.listenTo(this.model.get('videos'), 'reset', this.renderVideos);
+    this.listenTo(this.model, 'change:searchQuery', this.updateSearchUrl);
+    this.renderVideos();
+  },
+
+  setCurrentVideo: function () {
```

Comment:
This function should also be in the model as that is where `current_video` lives (as it should)